### DISCUSSION TITLE
idea for GUI

### DISCUSSION THREAD

**Discussion by Kay-1960** [saivert/pwvucontrol#22] [Ideas]
the best gui I ever see, was for Terratec EWX 24/96,
may be take some elements.
especially use ever dB, not %, for voöume and audio 
and never under no circumstances more than one position after decimal point

**Comment by saivert** [comment]
I'm not going to remove percentage from the display. I show both percentage and dB. This isn't software for audio geeks that only speak in sound pressure level or dB Full scale or something like that. It is a casual volume mixer.
